src/main/webapp/i18n/de/competency.json

```
@@ -14,14 +14,20 @@
             "optionalDescription": "Optionale Kompetenzen müssen nicht absolviert werden um den Lernpfad zu vervollständigen. Sie dienen entweder als zusätzliche Übung oder gehen über die erwarteten Inhalte des Kurses hinaus.",
             "course": "Kurs",
             "progress": "Fortschritt",
-            "progressDescription": "Der Prozentsatz an abgeschlossenen Vorlesungseinheiten und Übungen, die mit dieser Kompetenz verknüpft sind.",
-            "confidence": "Konfidenz",
-            "confidenceDescription": "Die durchschnittliche Bewertung in allen verknüpften Übungen im Verhältnis zu dem vom Lehrenden festgelegten Schwellenwert für die Beherrschung der Kompetenz.",
+            "progressDescription": "Der Fortschritt an Inhalten, die mit dieser Kompetenz verknüpft sind. Er basiert auf dem Prozentsatz an abgeschlossenen Vorlesungseinheiten und den erreichten Punkten in den Übungen.",
```

No strong opinion, does not need to be changed - can be read a bit more fluent in my opinion

```suggestion
            "progressDescription": "Der Fortschritt an Inhalten, die mit dieser Kompetenz verknüpft sind. Der Fortschritt basiert auf dem Prozentsatz an abgeschlossenen Vorlesungseinheiten und den erreichten Punkten in den Übungen.",
```

---

I prefer it as is. If others also prefer your suggestion, I'll change it